web/app/js/components/BaseTable.jsx

```diff
@@ -89,8 +90,13 @@ class BaseTable extends React.Component {
   generateRows = (tableRows, tableColumns, order, orderBy, filterBy) => {
     let rows = tableRows;
     let col = _find(tableColumns, d => d.dataIndex === orderBy);
+    let {defaultOrderBy} = this.props;
```

Comment:
TIOLI: instead of declaring `defaultOrderBy` here, you could add it to line 180 and pass it in to `generateRows()`

Reply:
I think rather than having a very long list of arguments to generateRows, I'd prefer to access things in the function where they're used. The parent render prop doesn't make use of defaultOrderBy at all.